### ISSUE TITLE
Display encapsulated subpatchers in Miraweb

### ISSUE THREAD

**Issue by tonetechnician** [Cycling74/miraweb#166]
Hey there! 

Amazing package. I've been loving it. Thanks to the dev for all the work!! 

I was just wondering if it's expected behavior for subpatchers (bpatchers) to not be displayed in mira? I've got a bunch of encapsulated objects and haven't been able to see them in my mira view. 

If it is, I think it would be super useful to get encapsulated subpatchers to display correctly in Mira.

**Comment by starakaj** [comment]
I agree, it would be extremely helpful. When we have some time to take a closer look at Mira, this will be at the top of our list for features to investigate. Thanks for making the FR!

**Comment by LasseMunk** [comment]
+1 to awesome FR
